hc/api/tests/test_list_checks.py

```diff
@@ -33,13 +33,15 @@ def get(self):
         return self.client.get("/api/v1/checks/", HTTP_X_API_KEY="abc")
 
     def test_it_works(self):
-        r = self.get()
+        resp = self.get()
         ### Assert the response status code
+        self.assertEqual(resp.status_code, 200)
 
-        doc = r.json()
+        doc = resp.json()
         self.assertTrue("checks" in doc)
 
         checks = {check["name"]: check for check in doc["checks"]}
+        import ipdb; ipdb.set_trace()
```

Comment:
@collinmutembei we must have committed that by mistake since we were doing front end tests